connectors/dune/README.md

```diff
@@ -0,0 +1,182 @@
+# Dune API Connector Example
+
+## Connector overview
+
+This connector retrieves query results from the [Dune API](https://dune.com/docs/api/) and syncs them into your Fivetran destination.  
+It allows you to execute one or multiple Dune queries, monitor their execution status, and fetch the final results into a unified destination table.
+
+The connector supports:
+- Running multiple queries sequentially.
+- Tracking execution status (`RUNNING`, `COMPLETED`, `FAILED`).
+- Rate limit handling (HTTP 429).
+- Incremental sync management using checkpointing.
+
+---
+
+## Requirements
+
+- [Supported Python versions](https://github.com/fivetran/fivetran_connector_sdk/blob/main/README.md#requirements)
+- Operating system:
+    - Windows: 10 or later (64-bit only)
+    - macOS: 13 (Ventura) or later (Apple Silicon [arm64] or Intel [x86_64])
+    - Linux: Ubuntu 20.04+, Debian 10+, or Amazon Linux 2+ (arm64 or x86_64)
+- A valid **Dune API Key**.
+- Pre-created **query IDs** in your Dune account.
+
+---
+
+## Getting started
+
+Refer to the [Connector SDK Setup Guide](https://fivetran.com/docs/connectors/connector-sdk/setup-guide) for setup instructions.
+
+---
+
+## Features
+
+- Executes one or many queries using Dune’s REST API.
+- Polls query execution status until completion or failure.
+- Fetches results from completed queries.
+- Handles **rate limiting (HTTP 429)** with exponential backoff.
+- Stores all query results in a single table for streamlined analysis.
+- Supports checkpointing for state tracking and recovery.
+- Logs progress, errors, and performance metrics for visibility.
+
+---
+
+## Configuration file
+
+The connector requires the following configuration parameters:
+
+```json
+{
+  "DUNE_API_KEY": "<YOUR_DUNE_API_KEY>",
+  "DUNE_QUERY_IDS": "<YOUR_DUNE_QUERY_IDS>"
+}
+```
+
+### Configuration parameters
+
+| Parameter | Description |
+|------------|-------------|
+| `DUNE_API_KEY` | Your Dune API key for authentication. |
+| `DUNE_QUERY_IDS` | Comma-separated list of Dune query IDs to execute and sync. |
+
+Note: Ensure that the `configuration.json` file is not checked into version control to protect sensitive information.
+
+---
+
+## Authentication
+
+The connector uses **API Key authentication** for secure access to the Dune API.
```

Comment:
PLease remove bolding across the file where it's not about UI elements such as tabs, menus, commands.
```suggestion
The connector uses API Key authentication for secure access to the Dune API.
```